src/Components/DrillList.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Maybe we could move this file to the `shared` subfolder ?

Reply:
sound good, done